samza-rest/src/test/java/org/apache/samza/monitor/TestLocalStoreMonitor.java

```diff
@@ -78,18 +82,22 @@ public void setUp() throws Exception {
   }
 
   @After
-  public void cleanUp() throws Exception {
+  public void cleanUp()  {
     // clean up the temp files created
-    FileUtils.deleteDirectory(taskStoreDir);
+    try {
+      FileUtils.deleteDirectory(taskStoreDir);
+    } catch (IOException e) {
+      // Happens when task store can't be deleted after test finishes.
+      Assert.fail();
```

Comment:
Exception in cleaning up the directory here can cause failures in actual test cases(since the directory will be counted in tests). 

Previously this was swallowed and hence we might not know if this delete failure has actually caused any assertion failure. 

If tests fails due to lack of permissions in cleaning up scaffold test directories, we will know that.